tests/unit/test.index.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+/* global -Promise */
+
+var Promise = require('es6-promise').Promise;
+var mockery = require('mockery');
+var nodemock = require('nodemock');
+
+var REMOTE_DEBUGGER_SOCKET = 'localfilesystem:/data/local/debugger-socket';
+
+module.exports = {
+
+  'firefoxOSDevice option should find ports forwarded to remote debugging socket on devices': function(test) {
+
+    var port = 9999;
+    var deviceId = '8675309';
+    var localAddress = 'tcp:'+port;
```

Comment:
Nitpick, but a space around the operator is easier to see. I'm surprised we don't have such a rule in the JSCS style check; I'll see about adding it.